ts/parts/Utilities.ts

```diff
@@ -2641,6 +2650,32 @@ const getStyle = H.getStyle = function (
     return style;
 };
 
+/**
+ * Get the defer as a number value from series animation options.
+ *
+ * @function Highcharts.getDeferTime
+ *
+ * @param {Highcharts.Chart} chart
+ *        The chart instance.
+ *
+ * @return {number}
+ *        The numeric value.
+ */
+const getDeferTime = H.getDeferTime = function (chart): number {
+
+    const seriesDelay = chart.series.map(function (series): number {
+        var seriesAnim = series.options.animation as any,
+            seriesDuration = seriesAnim.duration;
+
+        if (seriesAnim) {
+            return seriesAnim.defer ? seriesDuration + seriesAnim.defer : seriesDuration;
+        }
+        // If seriesAnim doesn't exist - return 0;
+        return 0;
+    });
+    return Math.max(...seriesDelay);
```

Comment:
Not a big deal, but we can replace this with `Math.max.apply(null, seriesDelay)` - avoiding `__spread` implementation in JS.

Perhaps even faster would be to use `forEach` + variable that is returned at the end:
```js
let maxDefer = 0;

series.forEach(s =>
  if (s.options.animation) {
      Math.max(maxDefer, (duration ? duration : 0) + (defer ? defer : 0));
  }
);

return maxDefer;
```